app/components/play-schedule/show/template.hbs

```diff
@@ -16,14 +16,14 @@
     </div>
 
     {{#if tracks}}
-      <div class="playlist-show-expand" {{action "togglePlaylistVisible"}}>
+      <div class="playlist-show-expand {{if this.isPlaylistVisible "is-expanded"}}">
         {{nypr-svg icon="left-arrow"}}
       </div>
     {{/if}}
   </div>
 </div>
 
-{{#if (and tracks this.isPlaylistVisible)}}
+{{#liquid-if (and tracks this.isPlaylistVisible)}}
   <div class="play-schedule-show-playlist">
```

Comment:
Are these working in the project currently? I was having problem with liquid-if, and am considering if we should use `ember-animated` or try to stick to CSS effects

Reply:
`liquid-if` works here. This is the only place I've used it. Is `ember-animated` better?

Reply:
But does it actually do a transition? `ember-animated` is the new rewrite of `liquid-fire`, so it's a little lower level in the way you write transitions but you can do more advanced things. I'm not sure how well the two play together right now